<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolAddWindow</class>
 <widget class="QDialog" name="ToolAddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tool Add Window</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_toolName">
     <property name="text">
      <string>Tool name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_toolName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_description">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_description"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_length">
     <property name="text">
      <string>Length</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_length"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_outerDiameter">
     <property name="text">
      <string>Outer diameter</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="spinBox_outerDiameter"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_innerDiameter">
     <property name="text">
      <string>Inner diameter</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="spinBox_innerDiameter"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_image">
     <property name="text">
      <string>Image</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_image">
     <item>
      <widget class="QLineEdit" name="lineEdit_image"/>
     </item>
     <item>
      <widget class="QPushButton" name="button_openFile">
       <property name="text">
        <string>Open File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_producerName">
     <property name="text">
      <string>Producer name</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="comboBox_producerName"/>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="button_clean">
       <property name="text">
        <string>Clean</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_input">
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
